tests/lib/rules/self-closing-comp.js

```diff
@@ -107,73 +107,84 @@ ruleTester.run('self-closing-comp', rule, {
   invalid: [
     {
       code: 'var contentContainer = <div className="content"></div>;',
+      output: 'var contentContainer = <div className="content"/>;',
```

Comment:
I believe that is the only ordering possible, because the `jsx-space-before-closing` rule does not produce a warning unless there is a self-closing tag. This obviously cannot happen if `self-closing-comp` is in violation, because that rule implies there is no self-closing tag. Therefore I think it is impossible for `jsx-space-before-closing` to produce an error on the same node as `self-closing-comp`, and the warning conditions for the two rules are mutually exclusive.